rotkehlchen/accounting/structures/evm_event.py

```diff
@@ -56,8 +56,16 @@ class EvmProduct(SerializableEnumNameMixin):
     GAUGE = auto()
 
 
-def get_tx_event_type_identifier(event_type: HistoryEventType, event_subtype: HistoryEventSubType, counterparty: str) -> str:  # noqa: E501
-    return str(event_type) + '__' + str(event_subtype) + '__' + counterparty
+def get_accounting_rule_identifier(
```

Comment:
Hmm why is this here? :sweat: 

You have already defined this in `accounting/rules.py`